clang/lib/Sema/Metafunctions.cpp

```diff
@@ -4420,6 +4420,25 @@ bool reflect_result(APValue &Result, Sema &S, EvalFn Evaluator,
   return SetAndSucceed(Result, APValue(RV));
 }
 
+bool is_nonstatic_member_function(ValueDecl *FD) {
+  if (!FD) {
+    return false;
+  }
+
+  if (dyn_cast<CXXConstructorDecl>(FD)) {
+    return false;
+  }
+
+  // todo: check that method is not special??
```

Comment:
This question is still relevant.

Do we want to prohibit trials of invoke for special members? E.g. for constructors we already have special brunch in logic.

Reply:
In side convestation we decided to not add any limitations for special members